apps/signer/README.md

```diff
@@ -42,9 +42,9 @@ meson install -C build_apps
 The executable is now located at `./my_installs/bin/signer.exe`
 
 ## Running
-Sign an MP4 file of an H264 video using the app
+Sign an MP4 file of an H264 video with recurrence 1 frames using the app
```

Comment:
I think we should refer to the API for more information on recurrence. Also, using a larger value, like 100, is a better choice since very few recordings sign every frame.